doc/conceptualOverview.dox

```diff
@@ -57,30 +56,161 @@ Poll inputs do not have the AccessMode::wait_for_new_data flag, and hence all re
+
+The \ref example_config_oven_xlmap "example xlmap mapping file" generates the following variable hierarchy:
+
+- /Configuration/heaterMode (redirects to to HEATER.MODE, read/write)
+- /Configuration/lightOn (redirects to BOARD.GPIO_OUT0, read/write)
+- /Controller/heatingCurrent (reditects to HEATER.CURRENT_SET, read/write)
+- /Controller/temperatureReadback (redirects to SENSORS.TEMPERATURE1, read only)
+- /Monitoring/heatingCurrent (redirects to HEATER.CURRENT_READBACK, read only)
+- /Monitoring/temperatureOvenTop (redirects to SENSORS.TEMPERATURE2, read only)
+- /Monitoring/temperatureOvenBottom (redirects to SENSORS.TEMPERATURE3, read only)
+- /Monitoring/temperatureOutside (redirects to SENSORS.TEMPERATURE4, read only)
+
+Since in ApplicationCore, all device registers are treated as unidirectional, read/write registers will be used in the write direction only. Hence, the first 3 variables will have the device as a consumer, receiving values either from an ApplicationModule or from the control system. The other variables will use the device as a feeder, so the device will provide values to the ApplicationModules and the control system.
+
+In case of our example, the device expects the application to poll the data (no registers support AccessMode::wait_for_new_data as the device does not support interrupts). Hence, the name of a push-type variable triggering the readout needs to be specified as a 3rd argument to the DeviceModule. This trigger variable will affect only registers which need a trigger and can even come from the device itself, in case it provides a data ready interrupt which shall be used to trigger the readout.
+
+For now, the only ApplicatioModule in our application is the Controller module, which connects to the /Controller/temperatureReadback as its input and to the /Controller/heatingCurrent as its output (variables/registers with identical name and path will be connected). Since /Controller/temperatureReadback is the push input of the Controller module, updates to this variable will trigger the Controller computations which will then write its result back to the /Controller/heatingCurrent register on the device.
+
+Because /Controller/temperatureReadback is a push-type input of the Controller module but a poll-type register of the device, a trigger is required to initiate the transfer. As specified, the variable /Timer/tick will be used for this.
+
+TODO: Device initialisation handler
+
+\section conceptualOverview_PeriodicTriggers Periodic Triggers
+
+So far, there is no source specified for the variabl /Timer/tick. In this case, the control system will automatically be used as a source. Since all control system variables are considered push type, they can be used as a trigger. Any write to this variable from the control system side will trigger now the device read outs, which could be realised e.g. as a button on a control system panel. While this might be useful in some cases, we cannot run a control loop like this.
```

Comment:
```suggestion
So far, there is no source specified for the variable /Timer/tick. In this case, the control system will automatically be used as a source. Since all control system variables are considered push type, they can be used as a trigger. Any write to this variable from the control system side will now trigger the device readouts, which could be realised e.g. as a button on a control system panel. While this might be useful in some cases, we cannot run a control loop like this.
```